Frameworks/SPMySQLFramework/Source/SPMySQLConnection.h

```diff
@@ -46,8 +46,11 @@
 	BOOL useSocket;
 	NSString *socketPath;
     
-    //Special connection settings
-    BOOL allowDataLocalInfile;
+  //Special connection settings
+  BOOL allowDataLocalInfile;
+    
+  // Clear text plugin
+  BOOL enableClearTextPlugin;
 
 	// SSL connection details
```

Comment:
Spacing might have got a little funky here?

Reply:
Yeah.. I'll update that. I think my editor was not setup correctly with tabs/spaces

Reply:
Fixed, looks like the allowDataLocalInFile was added with spaces instead of tabs.. I replaced it with tabs in a new commit so its consistent with the file.